event_mail/models/event.py

```diff
@@ -10,8 +8,7 @@ class EventEvent(models.Model):
 
     @api.model
     def _default_event_mail_template_id(self):
-        return self.env['ir.values'].get_default(
-            'event.config.settings', 'event_mail_template_id')
+        return self.env.user.company_id.event_mail_template_id
```

Comment:
You should use the default `company_id` value (that can be different than the user one).

Reply:
I don't understand this. Why would user from company A want a default for company B? :thinking:

Reply:
We don't know, but let the ORM do their job. If field `company_id` is set by default to any other value, the same should be used here. Any way, don't think only on different companies. Think on parent + child companies for example.

Reply:
But are you talking about the `company_id` default value in `event.event`, or in `res.users`?

Reply:
In `event.event`. Quick code:
```python
return self._default_get('company_id').event_mail_template_id
```

Reply:
I understood you were talking about the default company for the database, not for the event. You're right, thanks! :blush:

Reply:
Ok, changed :)